cast/tests/e2e/main.rs

```diff
@@ -1 +1,2 @@
+mod helper_functions;
 pub(crate) mod helpers;
```

Comment:
I think we should create a command to invoke all tests at once

Reply:
can you clarify this, please? I'm not sure I follow

Reply:
My bad. I forgot there is `cargo test` 🤦